src/Samples/Toolkit.SampleApp.Maui/Samples/BasemapGalleryAppearanceSample.xaml.cs

```diff
@@ -19,16 +19,16 @@ private async void Button_Add_Item(object? sender, EventArgs e)
             item.Name = "With Thumbnail";
             item.Tooltip = Guid.NewGuid().ToString();
             item.Thumbnail = new Esri.ArcGISRuntime.UI.RuntimeImage(new Uri("https://www.esri.com/content/dam/esrisites/en-us/home/homepage-tile-arcgis-collaboration.jpg"));
-            Gallery.AvailableBasemaps.Add(item);
+            Gallery.AvailableBasemaps?.Add(item);
```

Comment:
Might be better to just avoid all this code if it's null but starting the method with
`if (Gallery.AvailableBasemaps is null) return;`